sql/migrate/migrate.go

```diff
@@ -373,6 +397,117 @@ type templateFile struct {
 // Name implements the File interface.
 func (f *templateFile) Name() string { return f.n }
 
+// Filename of the migration directory integrity sum file.
+const hashFile = "atlas.sum"
+
+// HashFile represents the integrity sum file of the migration dir.
+type HashFile []struct{ N, H string }
+
+// HashSum reads the whole dir, sorts the files by name and creates a HashSum from its contents.
+func HashSum(dir Dir) (HashFile, error) {
+	var hs HashFile
+	h := sha256.New()
```

Comment:
```suggestion
	var (
		hs HashFile
		h = sha256.New()
	)
```
Just a suggestion, but I don't like to mix different declaration types in the same block